config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -107,6 +107,14 @@ spec:
               goflowkube:
                 description: GoflowKube contains settings related to goflow-kube
                 properties:
+                  healthPort:
```

Comment:
This changes should also be done to bundle/manifests/flows.netobserv.io_flowcollectors.yaml

Reply:
Do we need to add it manually? Didn't find any makefile target that regenerated those manifests

Reply:
Nevermind, I copied the additions manually.